swift-pipeline

```diff
@@ -158,13 +176,32 @@ if __name__ == "__main__":
         load_snapshot(f"{input}/{snapshot}")
         for input, snapshot in zip(args.input, args.snapshots)
     ]
-    run_names = [data.metadata.run_name for data in snapshots]
 
+    if args.run_names is None:
+        run_names = [data.metadata.run_name for data in snapshots]
+    else:
+        run_names = args.run_names
+
+    script_colours=["C0"]
```

Comment:
I suppose this would need to be removed should we go with the overriding stylesheet.